src/api/routes/block.ts

```diff
@@ -36,14 +39,33 @@ export function createBlockRouter(db: DataStore): RouterWithAsync {
     res.json(response);
   });
 
+  router.getAsync('/by_height/:height', async (req, res) => {
+    const height = parseInt(req.params['height']);
```

Comment:
need to handle the radix?

Reply:
I think it's okay as of a new-ish version of nodejs/ecmascript where the octal heuristic parsing was removed. But still, js so is sketch.. fixed